assets/javascripts/discourse/components/explorer-schema.js

```diff
@@ -48,21 +47,24 @@ export default Component.extend({
       });
     }
     return schema;
-  },
+  }
 
-  @discourseComputed("filter")
-  rfilter(filter) {
-    if (!isBlank(filter)) {
-      return new RegExp(filter);
+  get filteredTables() {
+    let tables = [];
+    let filter = this.filter;
+
+    const schema = this.transformedSchema;
```

Comment:
It comes down to personal preference but I usually don't create local vars from `this` props. Makes the code easier to understand w/o context. E.g. seeing `schema` on line 67/68 I'm thinking "what's that?" I scroll up to see it's just `this.transformedSchema`

Reply:
In this case we have to have a variable assignment for `this.filter` as we are manipulating the value without wanting to update the tracked value, but I agree and will update the `schema` const :)